v2/pkg/engine/plan/path_builder.go

```diff
@@ -138,34 +135,41 @@ func (p *PathBuilder) removeUnnecessaryFragmentPaths() (hasRemovedPaths bool) {
 }
 
 func (p *PathBuilder) printRevisitInfo() {
-	fmt.Println("Should revisit:", p.visitor.shouldRevisit())
-	fmt.Println("Has missing paths:", p.visitor.hasMissingPaths())
-	fmt.Println("Has fields waiting for dependency:", p.visitor.hasFieldsWaitingForDependency())
+	if p.visitor.shouldRevisit() {
+		fmt.Println("\tshould revisit")
+	}
+	if p.visitor.hasMissingPaths() {
+		fmt.Println("\thas missing paths")
+	}
+	if p.visitor.hasFieldsWaitingForDependency() {
+		fmt.Println("\thas fields waiting for dependency")
+	}
 
 	p.printMissingPaths()
 }
 
-func (p *PathBuilder) printPlanningPaths() {
-	debugMessage("\n\nPlanning paths:\n\n")
+func (p *PathBuilder) printPlanningPaths(run int) {
 	for i, planner := range p.visitor.planners {
-		fmt.Printf("\nPlanner id: %d\n", i)
-		fmt.Printf("Parent path: %s\n", planner.ParentPath())
+		fmt.Printf("\n\tRun #%d. Planner ID %d\n", run, i)
```

Comment:
please, remove tabs in this method. It will be hard to read when there are many planners
prefer smth like this
```
Planned paths on initial run #1:
	should revisit
	has missing paths
	has fields waiting for dependency

	Missing paths:
		query.accounts.complex

Run #1. Planner ID 0
Parent path: query
Datasource ID: id-1, name: id-1, hash: 11987355753863978781
Paths:
{"ds":11987355753863978781,"path":"query","shouldWalkFields":true,"pathType":"parent"}

```